clean_project/index.py

```diff
@@ -0,0 +1,48 @@
+"""
+Create a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
+
+from collections import defaultdict
+from clean_file import sanitize
+
+
+def index(config_path, index_path):
+    """
+    build a dictionary, given a configuration files
+    :param config_path: Yaml's path that contains a list of files that will be cleaned
+    :param index_path: Path of JSON file (indexing)
```

Comment:
i asked you to modify this docstring to hide the implementation a bit and make it more generic. plz fix!